docs/06_how-to-guides/02_multi-index/how-to-delete-data-from-a-multi-index-table.md

```diff
@@ -0,0 +1,30 @@
+## How to delete data from a multi index table
+
+### Preconditions it is assumed you already have a multi index table instance defined along with its mandatory primary index, otherwise take a look at the section [How to instantiate a multi index table](./how-to-instantiate-a-multi-index-table.md).
```

Comment:
Headings should never contain actionable data. This should be a list of preconditions (even if there is only one precondition)